scala-package/mxnet-demo/java-demo/Makefile

```diff
@@ -42,10 +41,12 @@ endif
 
 javademo:
 	(mvn install dependency:copy-dependencies package -Dmxnet.profile=$(SCALA_PKG_PROFILE) \
-		-Dmxnet.scalaprofile=$(SCALA_VERSION_PROFILE) \
-		-Dmxnet.version=$(MXNET_VERSION))
+		-Dmxnet.scalaprofile=$(SCALA_VERSION_PROFILE))
+
+java_ci_demo:
+	(mvn -Pci-nightly install dependency:copy-dependencies package -Dmxnet.profile=$(SCALA_PKG_PROFILE) \
```

Comment:
Can this not be just "mvn install"? Install will automatically build package because it is after it in the lifecycle and the copy-dependencies should occur automatically because it was added to the package phase.

Reply:
As far as I remember when doing this, copy-dependencies does not occur automatically. 
You either add it in the ```<build>``` in the pom file eg : https://github.com/apache/incubator-mxnet/blob/master/scala-package/mxnet-demo/scala-demo/pom.xml#L52, or you can do it via the maven command as shown here.

Reply:
Adding it in the build in the pom file should be the proper way of doing it. Otherwise, it makes it difficult if you do other maven commands such as test, install, or deploy because all the commands would need to work around copy-dependencies. In fact, we should probably try to remove all Makefiles throughout scala and use the maven commands alone.